pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_unpublishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,22 @@
+include::../attributes.adoc[]
+[id="unpublishing-a-module-in-pantheon_{context}"]
+= Unpublishing a module in {ProductShortName}
+
+Unpublish a module in {ProductShortName} from the Customer Portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
+* You are logged in.
+
+.Procedure
+
+. Navigate to link:{LinkToSearchPage}[Search page].
+
+. Click the name of the module you want to unpublish.
++
+If you don't see the module listed, search for the modules using the search box.
+
+. Click *Unpublish* next to the module version you want to unpublish.
++
+Once a module is unpublished, it is no longer available on the Customer Portal.
```

Comment:
But unpublish is not delete, right? It's still available in Pantheon? Maybe a short phrase to indicate that would be helpful, especially for the upstream community.

Reply:
Indeed, added a line indicating it is still available in Pantheon.

Reply:
I see this in 0a83c34.